modules/display.js

```diff
@@ -0,0 +1,37 @@
+class Displaysections {
+  constructor() {
+    this.form = document.querySelector('form');
+    this.listOfBooks = document.querySelector('.container');
+    this.list = document.getElementById('list');
+    this.addNew = document.getElementById('addNew');
+    this.contact = document.getElementById('contact');
+    this.addNewSction = document.getElementById('add-book');
+    this.listSection = document.getElementById('list-books');
+    this.contactSection = document.getElementById('contact-container');
+  }
+
+  singlePage = () => {
+    this.listSection.style.display = 'block';
+    this.contactSection.style.display = 'none';
+
+    this.list.addEventListener('click', () => {
+      this.listSection.style.display = 'block';
+      this.addNewSction.style.display = 'none';
+      this.contactSection.style.display = 'none';
+    });
+
+    this.addNew.addEventListener('click', () => {
+      this.listSection.style.display = 'none';
+      this.addNewSction.style.display = 'block';
+      this.contactSection.style.display = 'none';
+    });
+
+    this.contact.addEventListener('click', () => {
+      this.listSection.style.display = 'none';
+      this.addNewSction.style.display = 'none';
+      this.contactSection.style.display = 'block';
+    });
+  };
+}
+/* eslint-disable import/prefer-default-export */
+export const DisplaySections = new Displaysections();
```

Comment:
- [ ] Please don't disable linter errors instead fix them manually, It will help you to write more clean and professional code.